### DISCUSSION TITLE
article/coverview

### DISCUSSION THREAD

**Discussion by giscus** [thedavidweng/NotionNext#5] [Announcements]
# article/coverview

CoverView 是一个由 React 开发，不到 500kb 大小，在 GitHub
 开源的网页项目

https://blog.davidweng.tk/article/coverview

**Comment by tangly1024** [comment]
简直是为我量身定制😂😂😂

**Reply by thedavidweng** [reply]
确实，看哥们儿最近全都用的[这个](https://www.notion.so/image/https%3A%2F%2Fs3-us-west-2.amazonaws.com%2Fsecure.notion-static.com%2F10eb466a-599b-4513-bcbf-2331cf88078d%2Fpexels-photo-546819.webp?table=block&id=bee1fccf-a3bd-47a1-a7be-83cc71372d83&cache=v2)当封面
